linux_os/guide/system/permissions/restrictions/enable_nx/bios_enable_execution_restrictions/tests/wrong_value.fail.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+#
+# platform = multi_platform_rhel,multi_platform_fedora
+#
```

Comment:
You can remove these (line 2 as well) empty "comments":
https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#tests

Reply:
Just left them there for easier reading. Wanna create a styleguide?